packages/backend/src/database/migrations/20230718093901_add_dashbord_uuid_to_chart_table.ts

```diff
@@ -0,0 +1,21 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.schema.alterTable('saved_queries', (t) => {
+        t.uuid('dashboard_uuid')
+            .references('dashboard_uuid')
+            .inTable('dashboards')
+            .nullable()
+            .onDelete('CASCADE');
+        // make space id nullable
+        t.integer('space_id').nullable().alter({ alterType: false });
+    });
+}
+
+export async function down(knex: Knex): Promise<void> {
+    await knex.schema.alterTable('saved_queries', (t) => {
+        t.dropColumns('dashboard_uuid');
+        // make space id non nullable
+        t.integer('space_id').notNullable().alter({ alterType: false });
```

Comment:
we probably need to remove all the saved queries with space_id null to apply this

Reply:
good catch 👍